setup.py

```diff
@@ -1,36 +1,35 @@
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    from distutils.core import setup
+
```

Comment:
Why is this needed? Or: why has this not been an issue before? Is it due to the updated `setuptools` version?

Reply:
I thought of it mostly as flexibility. If for whatever reason the setuptools import failed, that an alternative setup was called.
But I see now that the way we have find_packages, makes this moot.